src/containers/BooksForm.js

```diff
@@ -0,0 +1,19 @@
+import Select from 'react-select';
+
+const BooksForm = () => {
+  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  const option = categories.map((category) => ({
+    value: category,
+    label: category,
+  }));
+  return (
+    <div className="booklist">
+      <form>
+        <input type="text" id="title" name="title" placeholder="Book Title" />
+        <Select option={option} />
```

Comment:
- [ ] [REQUIRED] No option is displayed in your form when I open it on my local machine. Please work on it.
![1](https://user-images.githubusercontent.com/58552748/139667840-50539ddf-6b6a-4e4b-b778-c2aa753b1e05.gif)